greenplumpython/__init__.py

```diff
@@ -1,8 +1,8 @@
-from .db import Database, database
-from .expr import get_type
-from .func import create_aggregate  # type: ignore
-from .func import create_array_function  # type: ignore
-from .func import create_function  # type: ignore
-from .func import aggregate, function
-from .op import operator
-from .table import Table, table, values
+from greenplumpython.db import Database, database
```

Comment:
I tried with this syntax, but had error, was it because of line 54 `html_static_path = ["_static"]` in conf.py?

Reply:
we need pip install . to install it as moudle

Reply:
Learnt a new thing